Migration script/openIMIS migration latest.sql

```diff
@@ -5849,3 +5985,6030 @@ NEXT_POLICY:
 	DEALLOCATE Cur
 END
 GO
+
+-- OTC-232: Integrate DB Stored Procedures: missing part
+IF OBJECT_ID('uspAPIGetClaimOverview', 'P') IS NOT NULL
+    DROP PROCEDURE [uspAPIGetClaimOverview]
+GO
+SET ANSI_NULLS ON
+GO
+SET QUOTED_IDENTIFIER ON
+GO
+
+-- 4/11/2019
+
+CREATE PROCEDURE [dbo].[uspAPIGetClaimOverview]
```

Comment:
I don't find this SP called in the REST API or WebApp. Can you check if this is actual used somewhere?

Reply:
Ok. I will check this and confirm this whether is somewhere in that apps